.storybook/spirit.theme.ts

```diff
@@ -33,6 +33,5 @@ export default create({
   // inputBorderRadius: 4,
 
   brandTitle: 'Spirit Design System',
-  brandUrl: 'https://spirit.lmc.eu',
```

Comment:
`https://spirit.design` ?

Reply:
IMHO it is better that the logo link goes to the root of the storybook. At least I would expect that when I click the top left logo it gets me to the website's root and not somewhere else.